src/controllers/collections.controllers.js

```diff
@@ -15,6 +27,12 @@ const postCollection = async (req, res) => {
     // for every measured value of the sensor, send to db
     await Promise.all(
       s.sensor_data.map(async (val) => {
+        // convert unix timestamp to date object
+        // since rover will be deployed quite often, only need to compare days rather than full date format
+        const dateObject = new Date(val.timestamp * 1000);
```

Comment:
Just a question, not saying it's wrong, but why do you have to multiply it by 1000?

Reply:
multiplying by 1000 converts unix to milliseconds